packages/cli/src/util/update-notifier/update-worker.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * This file is spawned in the background and checks npm for the latest version
+ * of the CLI, then writes the version to the cache file.
+ */
+
+// import fetch from 'node-fetch';
+// import { Agent as HttpsAgent } from 'https';
+// import { outputJSON } from 'fs-extra';
```

Comment:
Remove?

```suggestion
```

Reply:
Removed. You'll need to add it back when you port it to ESM.